00_Core/Scripts/Source/MME_Storage.psc

```diff
@@ -0,0 +1,93 @@
+Scriptname MME_Storage Hidden
+
+; Varables in StorageUtil
+; float
+;	MME.MilkMaid.BreastBase
+;	MME.MilkMaid.BreastBaseMod
+
+; must NiOverride be considered?
+; what happens if there is no such bone? (e.g. wrong skeleton or transformed?)
+function initializeActor(actor akActor) global
+	Debug.Trace("MME_Storage: Triggered initializeActor() for actor " + akActor.GetLeveledActorBase().GetName())
+
+	; <sanity check>
+	float BreastL = NetImmerse.GetNodeScale(akActor, "NPC L Breast", false)
+	float BreastR = NetImmerse.GetNodeScale(akActor, "NPC R Breast", false)
+	if (BreastL != BreastR)
+		string ActorName = akActor.GetLeveledActorBase().GetName()
+		Debug.Notification("[MilkModEconomy] " + ActorName + " has differently sized left and right breast!")
+		Debug.Notification("[MilkModEconomy] Asymetric breasts are not supported!")
+	endif
+	; </sanity check>
+
+	StorageUtil.SetFloatValue(akActor, "MME.MilkMaid.BreastBase", BreastL)
+	; <please review and remove this comment afterwards>
+	; might be a good idea to make sure a default value is always set but
+	; this might also introduce subtle breakage if someone has tampered
+	; with this variable before the mod initialises it
+	;  a) keep previous behaviour and leave it out
+	;  b) opt for 'safety-first' and set it
+	;  c) guard with hasfloatvalue() check?
+	; </please review and remove this comment afterwards>
+	StorageUtil.SetFloatValue(akActor, "MME.MilkMaid.BreastBaseMod", 0.0)
+	StorageUtil.SetFloatValue(akActor, "MME.MilkMaid.WeightBase", akActor.GetLeveledActorBase().GetWeight())
+endfunction
+
+function deregisterActor(actor akActor) global
+	Debug.Trace("MME_Storage: Triggered deregisterActor() for actor " + akActor.GetLeveledActorBase().GetName())
+
+	StorageUtil.UnsetFloatValue(akActor, "MME.MilkMaid.BreastBase")
+	StorageUtil.UnsetFloatValue(akActor, "MME.MilkMaid.BreastBaseMod")
+	StorageUtil.UnsetFloatValue(akActor, "MME.MilkMaid.WeightBase")
+	endfunction
+
+float function getBreastsBaseadjust(actor akActor) global
+	Debug.Trace("MME_Storage: Triggered getBreastsBaseadjust() for actor " + akActor.GetLeveledActorBase().GetName())
+	return StorageUtil.GetFloatValue(akActor, "MME.MilkMaid.BreastBaseMod")
```

Comment:
i think i should be 
return StorageUtil.GetFloatValue(akActor, "MME.MilkMaid.BreastBaseMod", missing = NetImmerse.GetNodeScale(akActor, "NPC L Breast", false))
otherwise i can probably cause an error if "MME.MilkMaid.BreastBaseMod" was not set for some reason